src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FinRemBulkScanFormTransformerFactory.java

```diff
@@ -12,23 +13,15 @@
 import static uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.BulkScanForms.FORM_A;
 
 @Component
-public class BulkScanFormTransformerFactory {
```

Comment:
Is it really worth extracting this? It seems not to be very re-usable code...

Reply:
I believe it'd be helpful for other teams to have a skeleton of solution. So that they just extend classes and do not copy and paste code.